lib/ext/heroku/command/addons.rb

```diff
@@ -251,7 +256,7 @@ def attach
       case response.status
       when 201
         display("done")
-        action("Setting #{response.body["name"]} vars and restarting #{app}") do
+        action("Setting #{response.body["name"]} and restarting #{app}") do
```

Comment:
This will say something like `Setting HEROKU_POSTGRESQL_CYAN and restarting #{app}` which is a bit meaningless or misleading. The original message at least implied it was "vars associated with this name". Maybe we should add a config_vars hash to the attachment serializer (I've considered this in heroku/api#3728 anyway)?